interface-definitions/service-mdns-repeater.xml.in

```diff
@@ -15,6 +15,31 @@
             <children>
               #include <include/generic-disable-node.xml.i>
               #include <include/generic-interface-multi.xml.i>
+              <leafNode name="ip-version">
+                <properties>
+                  <help>IP address version to use</help>
+                  <valueHelp>
+                    <format>_ipv4</format>
+                    <description>Use only IPv4 address</description>
+                  </valueHelp>
+                  <valueHelp>
+                    <format>_ipv6</format>
```

Comment:
And here

Reply:
As stated [above](https://github.com/vyos/vyos-1x/pull/2307#discussion_r1340336013). This is to make CLI show literal `ipv6` instead of `<h:h:h:h:h:h:h:h>`.